appinfo/info.xml

```diff
@@ -49,9 +49,11 @@ include text processing providers to:
 	<screenshot>https://github.com/nextcloud/assistant/raw/main/img/screenshot3.jpg</screenshot>
 	<background-jobs>
 		<job>OCA\TpAssistant\Cron\CleanupImageGenerations</job>
+		<job>OCA\TpAssistant\Cron\CleanupAssistantTasks</job>
 	</background-jobs>
 	<commands>
 		<command>OCA\TpAssistant\Command\CleanupImageGenerations</command>
+		<command>OCA\TpAssistant\Command\CleanupAssistantTasks</command>
```

Comment:
Everytime I read TpAssistant, I think of this:

![image](https://github.com/nextcloud/assistant/assets/986878/59467f25-49f8-4d98-aa8b-a2c34112d135)